pints/_mcmc/_slice_rank_shrinking.py

```diff
@@ -0,0 +1,265 @@
+# -*- coding: utf-8 -*-
+#
+# Slice Sampling - Covariance Adaptive: Rank Shrinking
+#
+# This file is part of PINTS.
+#  Copyright (c) 2017-2019, University of Oxford.
+#  For licensing information, see the LICENSE file distributed with the PINTS
+#  software package.
+#
+from __future__ import absolute_import, division
+from __future__ import print_function, unicode_literals
+import pints
+import numpy as np
+
+
+class SliceRankShrinkingMCMC(pints.SingleChainMCMC):
+    """
+    Implements Covariance-Adaptive Slice Sampling by Rank Shrinking,
```

Comment:
Most/All editors will have an option somewhere to draw a line at 79 chars. Please use 79 chars - and nothing else - as the guidelines to decide where line breaks go

Reply:
Have corrected this